src/SprykerMiddleware/Zed/Process/Business/Stream/JsonRowWriteStream.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerMiddleware\Zed\Process\Business\Stream;
+
+use SprykerMiddleware\Shared\Process\Stream\WriteStreamInterface;
+
+class JsonRowWriteStream implements WriteStreamInterface
+{
+    protected const KEY_SIZE = 'size';
+
+    /**
+     * @var resource
+     */
+    protected $handle;
+
+    /**
+     * @var string
+     */
+    protected $path;
+
+    /**
+     * @var array
+     */
+    protected $data = [];
+
+    /**
+     * @var int
+     */
+    protected $position;
```

Comment:
Should be properly defaulted as per documented type and expected code flow.

Reply:
JsonRowWriteStream::open() set the value to 0. 

Row 70: $this->position = 0;

Reply:
Everything until constructor finish counts. If it is then still null, it would have to be declared as such.
In your case you can avoid this by properly defaulting it in the first place.